gdal/gcore/gdalclientserver.cpp

```diff
@@ -438,7 +438,9 @@ CPL_UNUSED
     const char* pszCWD)
 {
 #ifdef WIN32
+#ifndef RTC_WINDOWS_UNIVERSAL
```

Comment:
Would it make more sense to push this change upstream?  Specifically using the compiler provided preprocessor macros (I think it's `WINAPI_FAMILY_SYSTEM`?).

Reply:
I had suggested that too when chatting with @chri7325 but he preferred this way for now

Reply:
...also if we want to do upstream, we should probably do a lot more than just comment out parts of the code that we know our use of the API never hits, but is very likely others could hit.